src/api/app/helpers/webui/request_helper.rb

```diff
@@ -194,4 +194,9 @@ def request_action_header(action, creator)
     end.html_safe
   end
   # rubocop:enable Style/FormatStringToken, Style/FormatString
+
+  def review_request_reason(bs_request, review)
+    bs_request.request_history_elements.where(description_extension: review[:id]).pluck(:comment).first ||
+      'No reason given'
```

Comment:
`'No reason given'` is never returned because even if the comment is an empty string, it is still truthy.

This then displays nothing in the old and new UI

![old-ui](https://user-images.githubusercontent.com/1102934/53898790-14ee8400-4039-11e9-8429-e4564283e693.png)
![new-ui](https://user-images.githubusercontent.com/1102934/53898796-1750de00-4039-11e9-8823-fcccef9a202f.png)

Reply:
It should also be covered in the specs.

Reply:
Fixed and added a helper spec for this.